tests/testthat/test-carehomes-vaccination.R

```diff
@@ -31,15 +31,14 @@ test_that("there are no infections if everyone is vaccinated with a vaccine
   expect_true(all(s[, 2, ] == s[, 2, 1]))
 })
 
-test_that("Everyone moves to vaccinated and stays there if everyone quickly
-          gets vaccinated with a vaccine preventing 100% of acquisition
+test_that("Every susceptible moves to vaccinated and stays there if everyone
```

Comment:
Can you shrink these really long explanations so that they fit on one line, so that the test indentation is not so weird? I know this is an annoying thing, but it's the stuff that accumulates over time (You will need to tell Rstudio to re-indent after doing this, that will make your lines much easier to deal with as you'll save ~10 characters)

If you need to explain the test more you can use a comment - this small string is the thing printed when the test fails, so should not be a paragraph in any case (affects a few others I believe)